src/components/Snippet.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { Controlled as CodeMirror } from 'react-codemirror2';
+
+import 'codemirror/lib/codemirror.css';
+
+import Title from './common/Title';
+import Input from './common/Input';
+
+import '../styles/Snippet.styl';
+
+class Snippet extends Component {
+  constructor() {
+    super();
+    this.state = { isShowEmbed: false };
+    this.toggleEmbed = this.toggleEmbed.bind(this);
+  }
+  toggleEmbed() {
+    this.setState({ isShowEmbed: !this.state.isShowEmbed });
+    return !this.state.isShowEmbed;
```

Comment:
Hm, I wonder, why do we need return statement here? We don't use it. Is there any particular reason to keep it here?

Reply:
you are right, there is no need in return statement